projects/opendr_ws/src/perception/scripts/object_detection_2d_gem.py

```diff
@@ -143,30 +199,22 @@ def listen(self):
         rospy.loginfo("GEM node started!")
         rospy.spin()
 
-    def callback(self, msg_rgb, msg_ir):
+    def callback(self, msg_color, msg_ir):
         """
         Callback that process the input data and publishes to the corresponding topics
-        :param msg_rgb: input color image message
-        :type msg_rgb: sensor_msgs.msg.Image
+        :param msg_color: input color image message
+        :type msg_color: sensor_msgs.msg.Image
         :param msg_ir: input infrared image message
         :type msg_ir: sensor_msgs.msg.Image
         """
+        rospy.logwarn("Entered callback!")
```

Comment:
This logwarn looks like it is needed for debugging purposes. If that's the case please remove it as it will produce lot's of unnecessary logging for the end-user.